packages/now-python/now_init.py

```diff
@@ -56,13 +56,22 @@ def now_handler(event, context):
         conn = http.client.HTTPConnection('0.0.0.0', port)
         conn.request(method, path, headers=headers, body=request_body)
         res = conn.getresponse()
-        data = res.read().decode('utf-8')
 
-        return {
+        return_dict = {
             'statusCode': res.status,
             'headers': format_headers(res.headers),
-            'body': data,
         }
+
+        data = res.read()
+
+        try:
+            return_dict['body'] = data.decode('utf-8')
+        except UnicodeDecodeError:
+            return_dict['body'] = base64.b64encode(data).decode('utf-8')
+            return_dict['encoding'] = 'base64'
```

Comment:
Please add a test fixture for this new behavior

Reply:
Done, I believe.